internal/compliance/snapshot_poller/pollers/aws/ec2_vpc.go

```diff
@@ -234,80 +248,81 @@ func buildEc2VpcSnapshot(ec2Svc ec2iface.EC2API, vpc *ec2.Vpc) *awsmodels.Ec2Vpc
 		State:                       vpc.State,
 	}
 
-	ec2Vpc.SecurityGroups = describeSecurityGroupsVPC(ec2Svc, vpc.VpcId)
-	ec2Vpc.NetworkAcls = describeNetworkACLsVPC(ec2Svc, vpc.VpcId)
-	ec2Vpc.RouteTables = describeRouteTables(ec2Svc, vpc.VpcId)
-	ec2Vpc.FlowLogs = describeFlowLogs(ec2Svc, vpc.VpcId)
-	ec2Vpc.StaleSecurityGroups = describeStaleSecurityGroups(ec2Svc, vpc.VpcId)
+	var err error
+	ec2Vpc.SecurityGroups, err = describeSecurityGroupsVPC(ec2Svc, vpc.VpcId)
+	if err != nil {
+		return nil, err
+	}
+	ec2Vpc.NetworkAcls, err = describeNetworkACLsVPC(ec2Svc, vpc.VpcId)
+	if err != nil {
+		return nil, err
+	}
+	ec2Vpc.RouteTables, err = describeRouteTables(ec2Svc, vpc.VpcId)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Nit: I usually like to combine these into the same line:

```suggestion
	if ec2Vpc.RouteTables, err = describeRouteTables(ec2Svc, vpc.VpcId); err != nil {
		return nil, err
	}
```